pkg/smartcontract/parameter_test.go

```diff
@@ -252,6 +255,27 @@ var unmarshalJSONTestCases = []struct {
 		},
 		input: `{"type":"PublicKey","value":"03b3bf1502fbdc05449b506aaf04579724024b06542e49262bfaa3f70e200040a9"}`,
 	},
+	{
+		input: `{"type":"InteropInterface","value":"null"}`,
```

Comment:
Maybe `"null"` -> `null`? There is another string test below.

Reply:
Hmmm, I thought I've fixed that already. I'll check.

Reply:
It's another one :)

Reply:
Fixed.